<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsdDialog</class>
 <widget class="QDialog" name="OsdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSD PROPERTY</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>951</width>
     <height>651</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>540</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>540</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCameraName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Camera Name
Off/On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNoVideo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>No Video
Off/On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStatusBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Status Bar
Off/On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRecordStatus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Record Status
Off/On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGpsStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>GPS Status
Off/On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCameraInfo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Camera Info
Off/On</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
